src/Config.php

```diff
@@ -1128,6 +1134,7 @@ protected function getDefaults()
             'debug_enable_whoops'         => false, /** @deprecated. Deprecated since 3.2, to be removed in 4.0 */
             'debug_error_use_symfony'     => false,
             'debug_permission_audit_mode' => false,
+            'debug_local_domains'         => [],
```

Comment:
So the people that can't remember to run it off, are going to remember to configure this? Yeah, nah!